lms/djangoapps/pocs/models.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from student.models import CourseEnrollment, AlreadyEnrolledError
+from xmodule_django.models import CourseKeyField, LocationKeyField
+
+
+class PersonalOnlineCourse(models.Model):
+    """
+    A Personal Online Course.
+    """
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    display_name = models.CharField(max_length=255)
+    coach = models.ForeignKey(User, db_index=True)
+
+
+class PocMembership(models.Model):
+    """
+    Which students are in a POC?
+    """
+    poc = models.ForeignKey(PersonalOnlineCourse, db_index=True)
+    student = models.ForeignKey(User, db_index=True)
```

Comment:
Sorry, this is just my ignorance of the feature showing. I watched the demo, but I didn't realize students had a way of choosing which POC to be looking at while they're in a course.